drivers/backup/auth.go

```diff
@@ -30,6 +30,8 @@ var PxCentralAdminPwd string
 const (
 	// PxCentralAdminUser px central admin
 	PxCentralAdminUser = "px-central-admin"
+	// PxCentralAdminGivenName is the given name of the PxCentralAdminUser
+	PxCentralAdminGivenName = "pxcentraladmin"
```

Comment:
We can remove this since are going to use `preferred_name` now and it will be same as `PxCentralAdminUser`